tests/core/test_composite.py

```diff
@@ -450,23 +450,28 @@ def test_transform_filter(ant, sphere, airplane, tetbeam, inplace):
     nested = multi_from_datasets(airplane, tetbeam)
     multi.append(nested)
     multi.append(None)
+    for i, _ in enumerate(multi):
+        multi.set_block_name(i, str(i))
 
     NUMBER = 42
     transform = pv.Transform().translate(NUMBER, NUMBER, NUMBER)
     bounds_before = np.array(multi.bounds)
     n_blocks_before = multi.n_blocks
+    keys_before = multi.keys()
 
     # Do test
     output = multi.transform(
         transform, inplace=inplace, transform_all_input_vectors=False, progress_bar=False
     )
     bounds_after = np.array(output.bounds)
     n_blocks_after = output.n_blocks
+    keys_after = output.keys()
 
     assert (output is multi) == inplace
     assert [(block_in is block_out) == inplace for block_in, block_out in zip(multi, output)]
```

Comment:
Not directly related to this PR, but this looks like a truthy-object (always asserts True). Missing `all`
```suggestion
    assert all((block_in is block_out) == inplace for block_in, block_out in zip(multi, output))
```

Reply:
Good catch. It seems that we have more. Could you open new PR for this?
```bash
$ grep "assert \[" ./tests/**/*.py
./tests/core/test_composite.py:    assert [(block_in is block_out) == inplace for block_in, block_out in zip(multi, output)]
./tests/examples/test_dataset_loader.py:    assert [os.path.isabs(file) for file in path]
./tests/examples/test_dataset_loader.py:    assert [os.path.isabs(file) for file in path_loadable]
./tests/examples/test_dataset_loader.py:    assert [os.path.isfile(file) for file in path_download]
./tests/examples/test_dataset_loader.py:    assert [
./tests/examples/test_dataset_loader.py:    assert [
```

Reply:
I fixed it in #6588. I added you as co-author.